### PR TITLE
Adopt Collapsible Console Section

### PR DESCRIPTION
**Pull request by hashar**
# Link to GitHub repository

https://github.com/jenkinsci/collapsing-console-sections-plugin

# When modifying release permission

List the GitHub usernames of the users who should have commit permissions below:
- `@hashar`

### Release permission checklist (for submitters)
- [x] The usernames of the users added to the "developers" section in the .yml file are the same the users use to log in to [accounts.jenkins.io](https://accounts.jenkins.io/).
- [x] All users added have logged in to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/) once.
- [x] I have mentioned an [existing team member](https://github.com/orgs/jenkinsci/teams) of the plugin or component team to approve this request.

[tasklist]
### Reviewer checklist
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it.

### CODE REVIEW

**Comment by hashar** [comment]
If I got it right I have to mention the accounts under the `developers` entry: @oleg_nenashev @mPokornyETM

**Comment by MarkEWaite** [comment]
Thanks for your willingness to adopt the plugin @hashar !

Since the plugin is up for adoption, the existing maintainers don't need to give their permission.  They've already acknowledged that they will accept new maintainers when they placed the plugin for adoption.

Usually we prefer that the adoption request mention at least one pending pull request that you intend to merge once you have adopted the plugin.  The ["Improve a plugin" tutorial](https://www.jenkins.io/doc/developer/tutorial-improve/) offers several improvements that will help this plugin.

**Comment by hashar** [comment]
I went to propose adopting the plugin after Jenkins removed support for Prototype.JS in 2.426.1. That prevents the plugin from fetching a table of content to be inserted in the sidebar since it uses `Ajax.Updater` which should be replaced by a modern `fetch()` call.  I have proposed that with https://github.com/jenkinsci/collapsing-console-sections-plugin/pull/26

Another small one is the compilation fails due to an unbound transitive dependency in the stack [INFRA-1310](https://issues.jenkins.io/browse/INFRA-1310) https://github.com/jenkinsci/collapsing-console-sections-plugin/pull/25. That keeps the exact same version of JQuery and is rather conservative.

Next I'd like to update the parent POM to benefit from all of its feature, notably styling with spotless and I feel adventurous I might add a few HtmlUnit tests :)

**Comment by timja** [comment]
@mPokornyETM any thoughts? given I see you are merging PRs there

**Comment by hashar** [comment]
@mPokornyETM thanks for the hint about `lockable-resources` plugin, I will definitely have a look at it!

For future pull requests made on that repo, should I go ahead and self merge them or are you interested in reviewing them? I am not too afraid of things being broken since I don't think that specific plugin is much used, but still I'd like to avoid merging dumb fixes :-]

**Comment by mPokornyETM** [comment]
> @mPokornyETM thanks for the hint about `lockable-resources` plugin, I will definitely have a look at it!
> 
> For future pull requests made on that repo, should I go ahead and self merge them or are you interested in reviewing them? I am not too afraid of things being broken since I don't think that specific plugin is much used, but still I'd like to avoid merging dumb fixes :-]

+ you does not need to check the lockable-resources plugin. just foloow this tutorial
https://www.jenkins.io/doc/developer/tutorial-improve/

+ about mergin, It is sitll good to wait for review. I will try to review all changes in time. Sometimes, there are no more (or very less) developer. In that case I will wait few days, and merge the PR with comment / label "merged without review". Just a hit for other new possible developers.

**Comment by hashar** [comment]
Great, thank you for you guidances @mPokornyETM ! That is yet again an excellent example of the collaboration happening in the Jenkins community :popcorn:
